### PR TITLE
GF#20727 Observation profile documentation; Patterns page

### PR DESCRIPTION
**Pull request by sliver007**

### CODE REVIEW

File: pages/_includes/StructureDefinition-qicore-adverseevent-introduction.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<tr>
+    <td>See the <a href="http://hl7.org/fhir/us/qicore/patterns.html">patterns</a> page for implementation and usage patterns.</td>
```
Comment (brynrhodes) [inline pages/_includes/StructureDefinition-qicore-adverseevent-introduction.html]: This path should be relative ("patterns.html"). Same applies to all the introduction content.

File: pages/_includes/StructureDefinition-qicore-allergyintolerance-introduction.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<tr>
+    <td>See the <a href="http://hl7.org/fhir/us/qicore/patterns.html">patterns</a> page for implementation and usage patterns.</td>
```
Comment (brynrhodes) [inline pages/_includes/StructureDefinition-qicore-allergyintolerance-introduction.html]: This path should be relative ("patterns.html")

File: pages/patterns.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+{% include header.html %}
+
+{% include container-start.html %}
+
+<div class="col-12">
+  <a name="patterns"> </a>
+  <h2>QI-Core - Patterns<a href="patterns.html#top" title="link to here" class="self-link"> <img src="target.png" width="20" class="self-link" height="20"/></a></h2>
+  <h4><a class="external free" href="http://hl7.org/fhir/us/qicore/StructureDefinition-qicore-observation.html" rel="nofollow">QI-Core - Observation</a> <a href="patterns.html#observation" title="link to here" class="self-link"> <img src="target.png" width="20" class="self-link" height="20"/></a></h3>
```
Comment (brynrhodes) [inline pages/patterns.html]: This path should be relative ("StructureDefinition-qicore-observation.html")

Comment (sliver007) [reply pages/patterns.html]: Done

**Pull request merged**